migration/sql-files/029-identities-foreign-key.sql

```diff
@@ -0,0 +1,2 @@
+-- Refactor Identities: add a foreign key constraint
+alter table identities add constraint identies_user_id_users_id_foreign foreign key (user_id) REFERENCES users (id);
```

Comment:
ok thanks for spotting the typo. I used the same naming pattern as I found on the `tracker_items` table, to be honest:

```
postgres=# \d tracker_items
                                      Table "public.tracker_items"
     Column     |           Type           |                         Modifiers
----------------+--------------------------+------------------------------------------------
...

Indexes:
    "tracker_items_pkey" PRIMARY KEY, btree (id)
    "tracker_items_remote_item_id_tracker_id_uni_idx" UNIQUE CONSTRAINT, btree (remote_item_id, tracker_id)
Foreign-key constraints:
    "tracker_items_tracker_id_trackers_id_foreign" FOREIGN KEY (tracker_id) REFERENCES trackers(id) ON UPDATE RESTRICT ON DELETE RESTRICT
```

but I'm fine with a shorter name, too.

However, I would not do a DELETE CASCADE here because I _think_ that we should to keep the identities even if the user is removed. Work items creator and assignees are associated with identities, and it would be weird to remove them, IMHO.
@alexeykazakov @aslakknutsen, what do you think about it ?